classroom.rb

```diff
@@ -0,0 +1,15 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :stidents
```

Comment:
- [ ] I think you have a typo right here, am not sure the program will understand what `stident` stands for 😄